packages/ripple-nuxt-tide/lib/core/config-loader.js

```diff
@@ -0,0 +1,212 @@
+const path = require('path')
+const fs = require('fs')
+const appDir = require('app-root-path')
+const kebabCase = require('lodash.kebabcase')
+
+// Groups of our configs
+const configGroups = {
```

Comment:
Not critical, but i think we should rethink this. It's not clear what these are or what they are doing.